coverage/.last_run.json

```diff
@@ -1,5 +1,5 @@
 {
   "result": {
-    "covered_percent": 100.0
+    "covered_percent": 98.57
```

Comment:
This drop is because `ApplicationHelper#render_markdown` is no longer (indirectly) covered. It wasn't _really_ tested before, but a quick spec might be worthwhile — I'd hate to loose that coveted 100%.

Reply:
Yep, I'd prefer not to be _that_ guy who's the reason for coverage not being 100% lol.